components/breadcrumb/index.en-US.md

```diff
@@ -0,0 +1,32 @@
+---
+category: Components
+type: Navigation
+title: Breadcrumb
+---
+
+Breadcrumb displays the current page in the system hierarchy position, and can back to previous.
```

Comment:
A breadcrumb displays the current location within a hierarchy. It allows going back to states higher up within the hierarchy.